GameServerLib/Logic/ObjectManager.cs

```diff
@@ -380,5 +381,27 @@ public bool TeamHasVisionOn(TeamId team, GameObject o)
 
             return false;
         }
+
+        public int CountUnitsAttackingUnit(AttackableUnit target)
+        {
+            int count = 0;
+            try
+            {
+                var enemyUnits = GetObjects();
+                var team = target.Team.GetEnemyTeam();
+                count = enemyUnits.Count(x =>
+                    x.Value is ObjAiBase aiBase &&
+                    aiBase.Team == team &&
+                    !aiBase.IsDead &&
+                    aiBase.TargetUnit != null &&
+                    aiBase.TargetUnit == target
+                );
+            }
+            catch (Exception ex)
+            {
+
```

Comment:
What exception is this throwing?
We should log errors or remove the try catch if it isn't necessary.

Reply:
Fixed.